nixos/modules/services/networking/expressvpn.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+{
+  options.services.expressvpn.enable = mkOption {
+    type = types.bool;
+    default = false;
+    description = ''
+      Enable the ExpressVPN daemon.
+    '';
+  };
+
+  config = mkIf config.services.expressvpn.enable {
+    boot.kernelModules = [ "tun" ];
+
+    systemd.services.expressvpn = {
+      description = "ExpressVPN Daemon";
+      serviceConfig = {
+        ExecStart = "${pkgs.expressvpn}/bin/expressvpnd";
```

Comment:
Ideally this should be configurable

Reply:
I don't think it's required for now. I'll add an option to configure package if there's ever a separate `expressvpn` package.

Reply:
Its not about different packages but overlays which is also not that important. For example this is important for systemd because otherwise you trigger a mass rebuild.